vision of the modpack/breakdown/general.md

```diff
@@ -0,0 +1,121 @@
+
+Circuit tiers are relatively well defined but don’t really correspond to voltage tiers in a one-to-one way (a machine may have an LV hull and take an MV circuit, and in HV you build IV circuits).
+
+Another common theme is materials that are hard to get in one tier (for example, only available as small ore, need strange processing, etc) become easier to obtain in a later tier. Tungsten for example. However, it is likely that  the “easier” path needs an appropriate line - you can get small amounts of platinum but you’re going to need a platline eventually.
+
+Tier materials are used to gate content, by requiring a titanium plate, for example, a Thaumcraft item can be restricted until you can obtain at least some titanium.
+</div>
+
+# 3. What should be Magic?
+<div align="justify">
+Magic is concepts and mechanisms tailored under a magic theme. Independent of how the code works in the background, these mods should have a magic feel and look. Instead of the “tech” they focus more on utility than progressing through the tiers. Examples are armor, tools, survival, teleport and whatnot.
+
+Magic can assist in progressing through the tiers, but it should not be the primary gating mechanism of any tier. In simpler terms, one should be able to reach the end of the tech tree without doing any magic.
+
+Advancement in Tech should assist in the development of magic, e.g. a ZPM player should receive some kind of boost to speed up/catch up development in magic if he wanted to. But the boost in this direction should be somewhat limited. It should not eventually give birth to the meme of "Hold off doing magic until UHV when you get to one click unlock everything".
+
+Magic Mods should be kept relatively distinct, with optional cross-over instead of treating everything as a Thaumcraft Addon and requiring Infusion (as an example). However it is nice to have everything integrated into the Thaumcraft research mechanisms. 
+</div>
+
+# 4. What should be space exploration?
+<div align="justify">
+
+Currently you mainly go to space to get plans for the next rocket, to get ores and to pump certain fluids and gases. Only a few planets have additional interesting mechanics, like obtaining certain extraterrestrial plants.
+
+Playing through the pack without space exploration is not an officially supported game mode. But we do not stand in the way of contributions to the pack to make it doable as long as they don't shift the progression of the pack. That means such alternatives need to be clearly more difficult/expensive than the main rocket progression.
+</div>
+
+# 5. What is the place of combat in the pack?
+<div align=justify>
+The main focus of combat in the pack is in the early game. The world is harsh and dangerous and you need to defend yourself and work on becoming strong yourself. There many defensive and offensive options for the player to invest into and, if you do so, after a few tiers regular mobs should no longer be a concern.
```

Comment:
Typo: There are many ....

Reply:
The main focus of combat in the pack is in the early game. The world is harsh and dangerous and you need to defend yourself and work on becoming strong. There are many defensive and offensive options to invest in and, if you do so, after a few tiers regular mobs should no longer be a concern.